e_imports.go

```diff
@@ -0,0 +1,25 @@
+// Copyright 2022 Gravitational, Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package teleport
+
+// Hold a few imports done exclusively by e/, so tidying that doesn't have
+// access to it (like Dependabot) doesn't wrongly remove them.
+// Any import that is present only in this file, but not in e/, can be safely
+// removed.
+import (
+	_ "github.com/go-piv/piv-go/piv"     // hold for e/
+	_ "github.com/gravitational/license" // hold for e/
+	_ "gopkg.in/check.v1"                // hold for e/
```

Comment:
This is a test-only dependency AFAIK. I don't think we need to import it.

Reply:
Go doesn't distinguish dependencies in that sense. Keeping it here seems correct; it's akin to keeping testify in the list.

Reply:
I don't mind keeping it here, but Go does distinguish between test and non-test dependencies. All of them need to be listed in `go.mod` but test-only dependencies are not included in the final build if they are not needed by any non-test module. You can list them by using `go list -f '{{join .TestImports " "}}' ./...`
I think that the simplest solution would be to add some build tag different than `ignore` as `go mod tidy`.

Reply:
Added `eimports` tag. Good call.